### PR TITLE
Fix for Italian original sentence dataset

### PR DESCRIPTION
**Pull request by Mte90**
@Sav22999 waiting for a check by him

I opened the pr in the wrong repo (we usually do before on our fork) but anyway the changes are very few, based on https://github.com/mozillaitalia/voice-web/issues/35

### CODE REVIEW

File: server/data/it/frasi.txt [hunk 1]
```diff
@@ -1133,7 +1133,7 @@ Questo viene diviso iniziando da 2 e a seguire con il risultato delle divisioni
 Questi valori vengono passati al Select che verifica una condizione booleana.
 Tutto questo circuito è contenuto in un For Loop per avere un aggiornamento in tempo reale del grafico.
 Abbiamo sviluppato un progetto.
-Ci permette di conoscere attraverso un grafico la resistenza sviluppata in 100 millisecondi (ms) alla temperatura di 100°C.
+Ci permette di conoscere attraverso un grafico la resistenza sviluppata in 100 millisecondi (ms) alla temperatura di 100 gradi.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: maybe it's better `°C`->`gradi Celsius`, `°`->`gradi`

File: server/data/it/frasi.txt [hunk 2]
```diff
@@ -1536,7 +1536,7 @@ Spingeranno il petrolio verso la torre di trivellazione.
 Dal petrolio greggio si possono ottenere attraverso la distillazione cherosene, benzene, benzina, pece, asfalto e bitumi.
 Questa estrazione avviene tagliando la catena molecolare del petrolio che è molto lunga, portandolo a diverse temperature.
 Questa distillazione viene effettuata in colonne alte più di trenta metri.
-Il liquido viene scaldato ad una temperatura oltre i 340 °C in un forno.
+Il liquido viene scaldato ad una temperatura oltre i 340 gradi in un forno.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: the same of row n.1136

File: server/data/it/frasi.txt [hunk 3]
```diff
@@ -1547,7 +1547,7 @@ Questa frammentazione può avvenire attraverso catalizzatori o per via termica.
 Con reforming invece si tratta di modifiche alla catena per poter rendere i derivati antidetonanti.
 Con antidetonante si intende la capacità di resistere ad alte pressioni senza esplodere.
 Questa caratteristica è indicata dal numero di ottani.
-Questa tecnica viene attuata ad alta temperatura, 750 - 800 °C, o a bassa, 400 - 450°C.
+Questa tecnica viene attuata ad alta temperatura, 750 - 800 gradi, o a bassa, 400 - 450 gradi.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: the same of row n.1136

File: server/data/it/frasi.txt [hunk 4]
```diff
@@ -9614,21 +9616,21 @@ Infatti, il primo deve istruire quindi deve insegnare il saper stare in acqua, i
...
-Allenatore I° livello, può allenare gli agonisti fino alla categoria giovanile.
-Allenatore II° livello, può allenare, essenzialmente, qualunque categoria.
+Allenatore primo livello, può allenare gli agonisti fino alla categoria giovanile.
+Allenatore secondo livello, può allenare, essenzialmente, qualunque categoria.
 Il primo deve insegnare e migliorare la tecnica.
 Il secondo deve migliorare i tempi di esecuzione degli stili.
 Per poter divenire allenatore bisogna prima conseguire la qualifica da istruttore (allievo, poi quello base).
 Successivamente si potrà conseguire la qualifica da allenatore.
-Prima quella di I° livello, poi quella di II° livello).
+Prima quella di primo livello, poi quella di secondo livello).
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: Remove the ")" at the end of the sentence

File: server/data/it/frasi.txt [hunk 5]
```diff
@@ -11169,4 +11171,5 @@ Le estensioni elencate indicano il formato di default in cui le suite di produtt
 Il formato di MS Office è un formato chiuso e proprietario di Microsoft.
 Lo evolve attraverso le varie versioni di Office in modo che supporti le nuove funzionalità offerte.
 È possibile che documenti creati con versioni precedenti a quella con la quale vengono letti presentino degli errori.
-Free non come "free beer".
\ No newline at end of file
+Free non come "free beer".
+Dare il 100%.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: `%` ->`percento`

**Comment by Sav22999** [comment]
Other corrections:
- At row 4755 -> remove the `"` (`Ho pensato: "È probabile siano stati sabotati gli addetti alle pulizie.`)
- At row 4789 -> remove the `"` (`"Preparati giovane Space, adesso tutte le tue malefatte verranno a galla.`)
- At row 4792 -> remove the `"` (`Finirai come Neo in Matrix nell'interrogatorio con Mr. Smith".`)
- At row 5786 -> remove `kk` and `cch` -> (`È necessario in quest'epoca in cui, ad esempio, si usa sempre più spesso "kk" invece che "cch".`) -> replace sentence with: `È necessario in quest'epoca in cui si usa sempre più spesso.`
- At rows: 2879, 2880, 2910, 2911, 2912, 2913, 2914, 9782, 10021, 10022, 10048, 10225, 10577, 10578, 10582, 10941, 10942, 11137, 11138, 11139, 11144, 11145 (and 11175): replace `%` with `percento`
- At row: 220: replace `mozillians.org` with `sito web` (`Che hanno contribuito in modo significativo alle attività Locali e hanno ottenuto un vouch su mozillians.org.`)
- At row 313: `es.` -> `esempio` (remove abbreviation)
- At row 2975: `n.109/96` -> `numero 109 del 1996`
- At row 4694: `a.C.` -> `avanti Cristo`
- At row 4792: `Mr.` -> `il signor`
- At row 6088: `n.` -> `numero`
- At rows 10353, 10354 and 10370: `M.I.T.` -> `MIT` (consider as a word)
- At row 10382: `S.` -> `Steven`
- At row 10471: `comp.os.minix` -> `Minix`
- At row 10765: `MSN.com` -> "di Microsoft`
- At rows 165, 169, 574, 4141, 5776, 9658, 9852, 9921 and 11167: `/` -> ` oppure `
- At rows 9920, 9936, 9981, 9987, 9991, 9995, 9998, 10003 and 10005: `m` -> ` metri`
- At rows 469, 1396 and 1545: `e/o` -> `e`
- At rows 380, 10452, 10548, 10631, 10663 and 10887: `/` -> ` ` (replace with empty space)
- At row 2867: `109/1996` -> `10 del 1996`
- At row 2869: `4/10` -> `4 del 2010` and `50/10` -> `50 del 2010`
- At row 2947: `109/96` -> `109 del 1996`
- At row 1594: `1/3` -> `un terzo`
- At rows 1587, 1735: `cm` -> `centimetri`
- At row 1769: `km` -> `chilometri`
- At row 9921: `50m/100m` -> `50 metri oppure 100 metri`

File: server/data/it/frasi.txt (hunk 5 above)
Comment (Mte90) [reply server/data/it/frasi.txt]: I was thinking here to leave as reference for the % symbol

File: server/data/it/frasi.txt [hunk 6]
```diff
@@ -466,7 +466,7 @@ Conforme alla disciplina vigente.
 Ai fini della liquidazione dei buoni incassati.
 In assenza, non potrà, in alcun modo, vedersi liquidare il valore dei buoni eventualmente incassati.
 È, pertanto, vietato incassare i buoni generati attraverso il servizio da un soggetto diverso rispetto al titolare del buono.
-Il cui nome è riportato sullo stesso e/o a fronte della vendita di beni o servizi diversi.
+Il cui nome è riportato sullo stesso e o a fronte della vendita di beni o servizi diversi.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: Secondo me, qui, è meglio lasciare solo "o" oppure "e" (meglio quest'ultima): sono troppe vocali a sé stanti, sembrano typo.
Inoltre, la "o" è molto ricorrente, quindi togliendola da qui non crea una "carenza" 😄

File: server/data/it/frasi.txt [hunk 7]
```diff
@@ -1584,14 +1584,14 @@ E anche perché non erano stati trovati molti fossili all'epoca (1925) in quel c
 Nel '59 vennero scoperti altri fossili in Africa e ciò confermò che era il continente della culla degli esseri umani.
 Venne scoperto tempo dopo un altro ominide chiamato Australopithecus Afarensis.
 Femmina databile sui 3 e 3,8 milioni di anni fa.
-Che era alta non più di 110 cm e che avrebbe potuto pesare sui 23 kg.
+Che era alta non più di 110 centimetri e che avrebbe potuto pesare sui 23 chili.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: Credo vada meglio "chilogrammi", perché altrimenti manca l'unità di misura dicendo solo "chili" 😄

File: server/data/it/frasi.txt [hunk 8]
```diff
@@ -4138,7 +4138,7 @@ Della necessità di raccontare storie.
 Continua a leggere Oltre ogni limite.
 Io vivo per l'attimo di silenzio che c'è prima dell'inizio di uno spettacolo o di un concerto o di una gara.
 È un momento di concentrazione massima.
-Non sfugge nulla, né al pubblico né agli attori/atleti/performers.
+Non sfugge nulla, né al pubblico né agli attori atleti e performers.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: Mettere una "," tra "attori" e "atleti"

File: server/data/it/frasi.txt [hunk 9]
```diff
@@ -4691,7 +4691,7 @@ L'arcaico "tu sorella", o sorellam tua (sì, non è il termine latino esatto, ma
 Veniva spesso anche usato come malaugurio per auspicare a qualcuno di avere in casa una militante.
 La stessa accezione negativa può essere estesa anche a "tu zia", in questo caso il malaugurio è di avere una zia sovversiva.
 Tutti i peggiori avvenimenti di Roma si dice che furono progettati da questa setta segreta, la setta delle Toga Nigra.
-Dall'incendio di Roma del 64 a.C.
+Dall'incendio di Roma del 64 avanti cristo.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: Cristo (lettera maiuscola)

File: server/data/it/frasi.txt [hunk 10]
```diff
@@ -5783,7 +5783,7 @@ Nella prima edizione era stata usata la chiave della grammatica.
 Nella seconda la punteggiatura.
 Questa volta, un argomento senz'altro più divertente: l'italiano delle canzoni.
 Sensibilizzare sull'uso corretto della lingua.
-È necessario in quest'epoca in cui, ad esempio, si usa sempre più spesso "kk" invece che "cch".
+È necessario in quest'epoca in cui, ad esempio, si usa sempre più spesso quest cosa.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: typo. quest->questa

File: server/data/it/frasi.txt [hunk 11]
```diff
@@ -10350,8 +10350,8 @@ Richard Stallman ama definirsi l'ultimo degli hacker.
 Inizialmente si iscrive presso la prestigiosa università di Harvard.
 Stallman viene spinto dalla sua passione per i calcolatori.
 È insofferente verso burocrazia, assenza di cooperazione e di scambio culturale.
-Decide di trasferirsi presso il laboratorio di Intelligenza Artificiale del M.I.T.
-M.I.T è acronimo di Massachusetts Institute of Technology.
+Decide di trasferirsi presso il laboratorio di Intelligenza Artificiale del Mit.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: "acronimo", lasciare tutto in maiuscola "Mit"->"MIT"

File: server/data/it/frasi.txt [hunk 12]
```diff
@@ -10350,8 +10350,8 @@ Richard Stallman ama definirsi l'ultimo degli hacker.
...
+Mit è acronimo di Massachusetts Institute of Technology.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: Come sopra

File: server/data/it/frasi.txt [hunk 13]
```diff
@@ -10762,7 +10762,7 @@ Per il 2010.
 Del successore di Vista, ovvero Windows 7.
 Microsoft sta inoltre cercando di allargare la sua quota di mercato per quanto riguarda le applicazioni Internet.
 Da alcuni anni punta in questa prospettiva sul software di messaggistica istantanea MSN Live Messenger.
-Windows Live Messenger si appoggia al portale MSN.com e al servizio di posta gratuito di Hotmail.
+Windows Live Messenger si appoggia al portale MSN e al servizio di posta gratuito di Hotmail.
```
Comment (Sav22999) [inline server/data/it/frasi.txt]: La pronuncia di "MSN" sarebbe "Emme esse enne", per questo credo vada meglio "di Microsoft"
(mentre, ad esempio, "MIT" lo si pronuncia (in genere) così come è scritto)

**Comment by Sav22999** [comment]
👍 perfect now, in my opinion.

**Comment by phirework** [comment]
Hi @Mte90 and @Sav22999, thank you both for your hard work on this PR! We don't accept localization content in `voice-web` directly, since this project use the Mozilla's Pontoon system to manage and merge localizations.

You can find the page for Italian here: https://pontoon.mozilla.org/it/common-voice/

Once you've submitted your changes, a community reviewer will check and approve them, and then they will be auto-incorporated into the website in our next deployment.

Thanks!

**Comment by Mte90** [comment]
This are not sentences for pontoon, those are the sentences for the CV
dataset like other pr to this repo.

Il lun 25 nov 2019, 19:54 JZ <notifications@github.com> ha scritto:

**Comment by nukeador** [comment]
Hi there.

For sentences, please remember we only accept PRs with removals, and corrected sentences should go through the sentence collector to ensure automatic filters are applied.

https://common-voice.github.io/sentence-collector/#/

If you change this PR to just remove sentences we will be able to merge.

Thanks!

**Comment by Mte90** [comment]
We fixed stuff that cannot be detected by filters in the first dataset that we did before the sentence collector, this doesn't involve sentence collector or wikipedia

**Comment by nukeador** [comment]
I know, but please understand this process is like this because we can't review ourselves every locale doing a PR, that's why we only accept removal PRs and ask for the modified sentences to be sent through the sentence collector, so the process there is automated.

Thanks!

**Pull request closed without merging**